modules/database/src/ioc/db/dbChannel.h

```diff
@@ -42,8 +42,8 @@ typedef struct evSubscrip {
     void                    *user_arg;
     struct event_que        *ev_que;
     db_field_log            **pLastLog;
-    unsigned long           npend;  /* n times this event is on the queue */
-    unsigned long           nreplace;  /* n times replacing event on the queue */
+    unsigned long           npend;  /** \brief n times this event is on the queue */
```

Comment:
This will actually comment the item after the comment. To comment `npend` you could either put the doc comment above, or use `/**<` or `//!<` doc comments